component/button/button.html

```diff
@@ -0,0 +1,103 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="../../main.css">
+    <link rel="stylesheet" href="button.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <div class="container">
+
+        <div class="row">
+            <p>Primary Button</p>
+            <button type='button' class='btn btn-primary'>Click</button>
+        </div>
+        <div class="row">
+            <p>Secondary Button</p>
+            <button type='button' class='btn btn-secondary'>Click</button>
+
+        </div>
+        <div class="row">
+            <p>Icon Button</p>
+            <button type='button' class='btn btn-primary'>
+                <i class="far fa-address-card"></i>   
+            Generate ID
+           </button>
+        </div>
+
+        <div class="row">
+            <p>Link Button</p>
+
+            <button type='button' class='btn btn-link'>Click</button>
+      
+        </div>
+
+
+        <div class="row">
+            <p>Floating Action Button</p>
+            <button type='button' class='btn-floating'>
+                <i class="far fa-bell"></i> 
+               </button>
+            
+        </div>
+
+
+        <div class="row">
+            <p>Outline Button</p>
+            <button type='button' class='btn-outline-primary btn'>
+                Outline button
+               </button>
+            
+        </div>
+
+
+    </div>
+
+
+
+    <!-- <button type='button' class='btn'>Click</button>
+
+
+
+
+
+
+
+
+   <button type='button' class='btn btn-primary'>
+    <i class="far fa-bell"></i> 
+   Alert Me
+   </button>
+
+
+
+
+
+   <button type='button' class='btn-floating'>
```

Comment:
try to be consistent with line spacing, you can use code formatter like prettier.

Reply:
ok I will correct the mistakes.